draft-demarco-oauth-status-attestations.md

```diff
@@ -369,8 +372,25 @@ MUST demonstrate the proof of possession of the Digital Credential to
 the Credential Issuer.
 
 The proof of possession is made by enclosing the Digital Credential in an
-object (JWT) signed with the private key that its related public key is
-referenced in the Digital Credential.
+object (JWT or CWT) signed with the private key that its related public key is referenced in the Digital Credential.
```

Comment:
I would rephrase saying using the same confirmation method referenced ... 

therefore using the elements defined in RFC7800

Reply:
The commit https://github.com/peppelinux/draft-demarco-oauth-status-attestations/pull/37/commits/5f23e8e156d732e1d2fbad89b81b62b7952f69fd aims to resolve this comment.